index.html

```diff
@@ -0,0 +1,215 @@
+                        education and better living conditions.</p>
+                </div>
+
+                <q>The greatness of humanity is not in being human, but in being
+                    humane. – Mahatma Gandhi
+                </q>
+
+            </section>
+
+        </div>
+        <section class="values">
+            <h3>Our Values</h3>
+            <hr>
+            <div class="values-container">
+                <div class="bg">
+                    <ul class="values-reasons">
+                        <li><img
+                                src="/first-capstone-project/assets/images/integrity.png" /></li>
+                        <li><p>Integrity</p>
+                        </li>
+                        <li>
+                            <p>We focus on the entire customer lifecycle.</p>
+                        </li>
+                    </ul>
+                </div>
+                <div class="bg">
+                    <ul class="values-reasons" id="values">
+                        <li>
+                            <img
+                                src="/first-capstone-project/assets/images/responsive.png" />
+                            <li><p>Responsive</p></li>
+                            <li>
+                                <p>To the needs of the children in our target
+                                    group
+                                </p>
+                            </li>
+
+                        </li>
+
+                    </ul>
+
+                </div>
+
+                <div class="bg">
+                    <ul class="values-reasons">
+                        <li>
+                            <img
+                                src="/first-capstone-project/assets/images/enriching.png" />
+                            <li><p>Enriching</p></li>
+                            <li>
+                                <p>
+                                    bettering lives through education and
+                                    other avenues
+                                </p>
+                            </li>
+
+                        </li>
+
+                    </ul>
+
+                </div>
+
+            </div>
+            <button type="button" class="donate-btn">Donate</button>
+
+            <div class="donate">
+                <img src="/first-capstone-project/assets/images/donate.png" />
+            </div>
+
+        </section>
+        <section class="projects">
+            <h3>Upcoming Events</h3>
+            <hr class="title-hr" />
+            <div class="events-list"></div>
+            <div class="more-btn" id="more-btn">
+                <p class="more">MORE</p>
+                <i class="arrow down"></i>
+
+            </div>
+            <div class="more-btn less-btn" id="less-btn">
+                <p class="more">LESS</p>
+                <i class="arrow up"></i>
+
+            </div>
+        </section>
```

Comment:
Great work done by making this section content be generated dynamically 🥇 , only some small styling adjustements to do 

- [ ] Increase font size of the title 
- [ ] reduce the size of the images 
- [ ] increase the margin of the container 

![Capture d’écran du 2023-08-02 00-37-07](https://github.com/NoelLincoln/first-capstone-project/assets/59707859/dc663b03-c947-4800-8479-46d81a1293b1)
![Capture d’écran du 2023-08-02 00-36-32](https://github.com/NoelLincoln/first-capstone-project/assets/59707859/8a6ff377-ed4c-44b5-8664-b13af5c73d40)